src/components/Search/Search.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+type Props = {
+  query: string;
+  setQuery: (target: string) => void
```

Comment:
Lifting state up using 👍

Reply:
but why argument called the target and not the value?

Reply:
you right, I fix that now